tests/foreman/cli/test_contentviews.py

```diff
@@ -315,6 +318,180 @@ def test_delete_cv_version(self):
+            u'id': cvv['id'],
+            u'to-lifecycle-environment-id': env[0]['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+
+        destination_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': destination_cv['id']})
+        result = ContentView.info({u'id': destination_cv['id']})
+        cvv = result.stdout['versions'][0]
+        result = ContentView.version_promote({
+            u'id': cvv['id'],
+            u'to-lifecycle-environment-id': env[1]['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+
+        result = ContentView.info({u'id': source_cv['id']})
+        self.assertEqual(result.stdout['content-host-count'], '0')
+
+        make_content_host({
+            u'name': gen_alphanumeric(),
+            u'organization-id': new_org['id'],
+            u'lifecycle-environment-id': env[0]['id'],
+            u'content-view-id': source_cv['id'],
+        })
+
+        result = ContentView.info({u'id': source_cv['id']})
+        self.assertEqual(result.stdout['content-host-count'], '1')
+        result = ContentView.info({u'id': destination_cv['id']})
+        self.assertEqual(result.stdout['content-host-count'], '0')
+
+        result = ContentView.remove({
+            u'name': source_cv['name'],
+            u'organization': new_org['name'],
+            u'environment-ids': env[0]['id'],
+            u'system-content-view-id': destination_cv['id'],
+            u'system-environment-id': env[1]['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentView.info({u'id': source_cv['id']})
+        self.assertEqual(result.stdout['content-host-count'], '0')
+        result = ContentView.info({u'id': destination_cv['id']})
+        self.assertEqual(result.stdout['content-host-count'], '1')
+
+    def test_remove_cv_version(self):
+        """@Test: Delete content view version through 'remove' command
+
+        @Feature: Content Views
+
+        @Assert: Content view version deleted successfully
+
+        """
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': new_cv['id']})
+        result = ContentView.info({u'id': new_cv['id']})
+        env = result.stdout['lifecycle-environments'][0]
+        cvv = result.stdout['versions'][0]
+        result = ContentView.remove_from_environment({
+            u'id': new_cv['id'],
+            u'lifecycle-environment-id': env['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+
+        result = ContentView.remove({
+            u'id': new_cv['id'],
+            u'content-view-version-ids': cvv['id'],
+            u'organization-id': new_org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentView.info({u'id': new_cv['id']})
+        self.assertFalse(result.stdout['versions'])
```

Comment:
better to check the length: `self.assertEqual(len(result.stdout['versions']), 0)`. This will give more meaning when someone else is reading the test.